include/rpm/rpmkeyring.h

```diff
@@ -23,7 +23,8 @@ extern "C" {
 typedef enum rpmKeyringModifyMode_e {
     RPMKEYRING_ADD	= 1,
     RPMKEYRING_REPLACE	= 2,
-    RPMKEYRING_DELETE	= 3
+    RPMKEYRING_DELETE	= 3,
+    RPMKEYRING_MERGE	= 4,
```

Comment:
This is missing API docs (see just above this enum def).

Reply:
In terms of semantics, I'm happy that `MERGE` acts like `ADD` if the certificate is not present.

With respect to the documentation, I think it will prevent some bugs if we add a note that "add" and "replace" are almost never what the caller actually wants and they should always almost prefer "merge."  Perhaps add some examples of when one ought to use "add" or "replace".  (I can't think of any sensible ones, TBH.)